codex_of_the_damned/templates/archetypes/contributions/creeping-sabotage.html

```diff
@@ -0,0 +1,140 @@
+	improvised_tactics=card("Improvised Tactics"),
+	shadow_boxing=card("Shadow Boxing"),
+	slam=card("Slam"),
+	swallowed_by_the_night=card("Swallowed by the Night"),
+	the_warrens=card("The Warrens"),
+	thrown_sewer_lid=card("Thrown Sewer Lid"),
+
+	a_b=link("/strategy/fundamentals", _("A&B"), _anchor="AB"),
+	bleed=link("/strategy/fundamentals", _("Bleed"), _anchor="bleed"),
+	bounce=link("/strategy/fundamentals", _("Bounce"), _anchor="bounce"),
+	combat=link("/strategy/combat", _("Combat")),
+	delay=link("/strategy/fundamentals", _("Delay"), _anchor="delay"),
+	finnish_politics=link("/strategy/archetypes", _("Finnish Politics"), _anchor="finnish_politics"),
+	intercept=link("/strategy/fundamentals", _("Intercept"), _anchor="stealth-and-intercept"),
+	range_management=link("/strategy/combat", _("Range management"), _anchor="range-management"),
+	rush=link("/strategy/archetypes", _("Rush"), _anchor="rush"),
+	s_ce=link("/strategy/combat", _("S:CE"), _anchor="defence"),
+	stealth=link("/strategy/fundamentals", _("Stealth"), _anchor="stealth-and-intercept"),
+	wake_and_unlock=link("/strategy/fundamentals", _("Wake and Unlock"), _anchor="wake_and_unlock"),
+
+	anson_guns=link("/archetypes/bibliodeque/anson-guns"),
+	nananimalism=link("/archetypes/nananimalism")
+%}
+<h2>Overview</h2>
+<p>
+	This deck uses neither bleed nor vote to deliver pool damage but uses {{ anarch_revolt }},
+	{{ army_of_rats }} and {{ creeping_sabotage }}. This way of ousting one's prey disables
+	a lot of usual defensive modules such as {{ bounce }} or {{ delay }}.
+</p>
+<p>
+	This strategy costs a bit of blood and those cards need to be defended. We go mid-cap
+	for speed and {{ a_b }}. The key here is having the upper hand in {{ combat }} hence the
+	use {{ POT }} as a core discipline of the deck as it is great at dealing high damage.
+</p>
+<p>
+	Nosferatus {{ img_clan_nosferatu }} are good at {{ ani }}, {{ obf }} and {{ pot }} disciplines,
+	{{ slag }} and {{ ryan }} for example. Some even have a built-in {{ rush }} action
+	({{ ebanezer_roush }} and {{ gustave_brunnelle }}) but they are not used because of their
+	higher capacity and the low presence of mid-cap {{ POT }} vampires in their grouping.
+</p>
+<p>
+	This {{ rush }} strategy leads to {{ finnish_politics }}. Its point is to use {{ rush }}
+	as a means of negociation with the other players. Played with {{ immortal_grapple }}, it
+	becomes a huge threat to {{ s_ce }} defensive modules. It is also the only defense against an
+	aggressive predator.
+</p>
+<p>
+	We go with {{ bums_rush}} and {{ haven_uncovered }} despite {{ ANI }} giving access to
+	{{ deep_song }}. As the point is to disable {{ bounce }}, it would go against that
+	objective because {{ bleed }} for 2 are nice to bounce.
+</p>
+
+<h2>Range Management</h2>
+<p>
+	{{ range_management }} is sometimes hard to deal with. Here are some {{ POT }} cards
+	that can be used and each one of them is a metacall answer:
```

Comment:
metal ? Is that a thing ?

Reply:
A `metacall` is a strategic choice made for an expected local metagame. Choosing one card over the others is a metacall